docs/en-us/user/references/metadata/introduction.md

```diff
@@ -0,0 +1,369 @@
+# Background
+There are close to [30 configurations](http://dubbo.apache.org/en-us/docs/user/references/xml/dubbo-service.html) in dubbo provider. Excluding registry center governance requirements, a large portion of configurations are used by the provider itself and do not need to be delivered to the consumer. This part of the data does not need to be written to the registry, but only needs to be persisted as key-value.
+There are also [20+ configurations](http://dubbo.apache.org/en-us/docs/user/references/xml/dubbo-reference.html) in dubbo consumer. In the registry center, only a few configurations such as application, version, group, ip, dubbo version are needed in the list of service consumers. Other configurations can also be persisted in key-value form.
+This data is registered into the registry in the service dimension, which leads to the expansion of data volume, and then causes the increased network overhead of the registry (such as zookeeper) and decreased performance.
+  
+In addition to the storage of the above configuration items, Dubbo service metadata information also needs to be stored. Metadata information includes service interface and method information of interface. This information will be used for service mock, service test.  
+
+
+
+# Goal
+
+The original data and metadata information in the registry center need to be stored in a separate key-value store, which can be DB, redis or other persistent storage. The core code supports zookeeper, redis(recommended) by default.
+
+The format of provider storage content is the storage after gsonization of org.apache.dubbo.metadata.definition.model.FullServiceDefinition.
+Consumer stores content in Map. Get parameter information from the URL written to the registry from the Consumer side. That is, get the Map with URL.getParameterMap() and store it after gsonization.
```

Comment:
Could you re-check this sentence ?
```
Get parameter information from the URL written to the registry from the Consumer side.
```

Reply:
I re-checked the original text and found that I did not fully understand the author's meaning before. I think the following translation is better:  

Consumer gets parameter information from the URL that it wrote to the registry and stores it in Map.